examples/webhook-signing/typescript-node-express/express-ts.ts

```diff
@@ -0,0 +1,76 @@
+import Stripe from 'stripe';
+import express from 'express';
+import bodyParser from 'body-parser';
+import env from 'dotenv';
+
+env.config();
+
+const stripe: Stripe = new Stripe(process.env.STRIPE_SECRET_KEY, {
+  apiVersion: '2019-12-03',
+  typescript: true,
+});
+
+const webhookSecret: string = process.env.STRIPE_WEBHOOK_SECRET;
+
+const app: express.Application = express();
+
+// Only use the raw body parser for webhooks.
```

Comment:
I think we should remove this block, it implies that you need it, but I think you don't – the `bodyParser.raw` call is enough (worth double-checking, I'm not 100% sure
)

Reply:
I think this was specifically to showcase that for all other routes we want to use the json parser. See Paul's original comment on that: https://github.com/stripe/stripe-node/pull/749#discussion_r363573568

Reply:
Yeah... if we did this:

```js
app.use(bodyParser.json());

app.post(
  '/webhook',
  bodyParser.raw({type: 'application/json'}),
  (req: express.Request, res: express.Response): void => {
```
would the app still work?

cc @adreyfus-stripe maybe you know?